docs/hardware/devices/heltec-automation/vision-master/index.mdx

```diff
@@ -0,0 +1,206 @@
+
+:::warning
+
+Do not proceed unless an antenna is connected to avoid possible damage to the device's radio.
+
+:::
+
+The following process will manually place the device into the Espressif Firmware Download mode:
+
+1. Unplug the device.
+2. Press and hold the BOOT button.
+3. Plug the device in.
+4. After 2-3 seconds, release the BOOT button.
+
+With the device now in the Espressif Firmware Download mode, you can proceed with flashing using one of the supported flashing methods. It's generally recommended to use the [Web Flasher](https://flasher.meshtastic.org/). You can select "Heltec Vision Master E290" from the device drop-down.
+
+### Pin Map
+
+![HT-VME290_PIN_MAP](/img/hardware/heltec/HT-VME290_PIN_MAP.webp)
+
+Image Source: [Heltec](https://resource.heltec.cn/download/HT-VME290/HT-VME290%20Pin%20map.png)
+
+### Resources
+
+There is an additional interface on the board for 2.4GHz wireless signals. When connecting an external antenna, the resistor needs to be rotated 90 degrees to the left. For more details, please check the [FAQ on Heltec's official website](https://heltec.org/project/vision-master-e290/).
+
+- Firmware file: `firmware-vision-master-e290-X.X.X.xxxxxxx.bin`
+- Purchase Links:
+  - [Heltec](https://heltec.org/project/vision-master-e290/)
+  - [AliExpress](https://www.aliexpress.com/item/1005007234361986.html)
+
+</TabItem>
+<TabItem value="vision_master_t190">
+
+## Vision Master T190
+
+- **MCU:**
+  - ESP32-S3R8 (WiFi & Bluetooth)
+- **LoRa Transceiver:**
+  - HT-RA62(Semtech SX1262)
+- **Frequency Options:**
+  - 433 MHz
+  - 470 - 510 MHz
+  - 863 - 870 MHz
+  - 902 - 928 MHz
+- **Connectors:**
+  - USB-C
+  - 2x1.25 lithium battery interface
+  - SH1.0-4P sensor interface
+  - Antenna:
+    - U.FL/IPEX antenna connector for LoRa
+    - Integrated 2.4 GHz PCB antenna, U.FL/IPEX antenna connector for 2.4 GHz.
+
+### Features
+
+- Onboard 1.90-inch TFT-LCD color screen
+- Boot, Reset, and USER switches
+- No GPS
+
+### Flashing
+
+To flash ESP32-S3 devices like the Vission Master T190, you typically need to place them in Espressif's firmware download mode. Use the "1200bps reset" button in the web flasher to do this. If this method does not work for any reason, you can follow the manual process below.
+
+:::warning
+
+Do not proceed unless an antenna is connected to avoid possible damage to the device's radio.
+
+:::
+
+The following process will manually place the device into the Espressif Firmware Download mode:
+
+1. Unplug the device.
+2. Press and hold the BOOT button.
+3. Plug the device in.
+4. After 2-3 seconds, release the BOOT button.
+
+With the device now in the Espressif Firmware Download mode, you can proceed with flashing using one of the supported flashing methods. It's generally recommended to use the [Web Flasher](https://flasher.meshtastic.org/). You can select "Heltec Vision Master T190" from the device drop-down.
+
+### Pin Map
+
+![HT-VMT190_PIN_MAP](/img/hardware/heltec/HT-VMT190_PIN_MAP.webp)
```

Comment:
Some of these files, even though they are webp, are _giant_. They should probably be scaled down in resolution so that they do not use as much space.

Again 🍺 nice to have not a blocker

Reply:
128kb is definitely acceptable for that resolution. Thanks for doing that